src/pages/settings/Wallet/ChooseTransferAccountPage.tsx

```diff
@@ -50,40 +52,73 @@ function ChooseTransferAccountPage({walletTransfer = {}}: ChooseTransferAccountP
         BankAccounts.openPersonalBankAccountSetupView();
     };
 
+    const [bankAccountsList] = useOnyx(ONYXKEYS.BANK_ACCOUNT_LIST);
+    const selectedAccountID = walletTransfer?.selectedAccountID;
+    const data = useMemo(() => {
+        const options = Object.values(bankAccountsList ?? {}).map((bankAccount) => {
+            const bankName = (bankAccount.accountData?.additionalData?.bankName ?? '') as BankName;
+            const bankAccountNumber = bankAccount.accountData?.accountNumber ?? '';
+            const bankAccountID = bankAccount.accountData?.bankAccountID ?? bankAccount.methodID;
+            const {icon, iconSize, iconStyles} = getBankIcon({bankName, styles});
+            return {
+                value: bankAccountID,
+                text: bankAccount.title,
+                leftElement: icon ? (
+                    <View style={[styles.flexRow, styles.alignItemsCenter, styles.mr3]}>
+                        <Icon
+                            src={icon}
+                            width={iconSize}
+                            height={iconSize}
+                            additionalStyles={iconStyles}
+                        />
+                    </View>
+                ) : null,
+                alternateText: `${translate('workspace.expensifyCard.accountEndingIn')} ${getLastFourDigits(bankAccountNumber)}`,
+                keyForList: bankAccountID?.toString(),
+                isSelected: bankAccountID?.toString() === selectedAccountID,
+                bankAccount,
+            };
+        });
+        return options;
+    }, [bankAccountsList, selectedAccountID, styles, translate]);
+
+    if (isLoadingOnyxValue(walletTransferResult)) {
```

Comment:
I don't see any way to trigger this block. @allgandalf Could you add a video to see the loading indicator in this page?